doc/changelog.txt

```diff
@@ -190,7 +191,7 @@ Fixes:
  - Fix using Set<Any>RulesParam with an incorrect type leaving that param set with a value of 0.
  - Fix COB SetMaxReloadTime receiving a value 10% smaller than it should
  - Fix builders not placing nanoframes from their maximum range
-
+-  Fix AllyTeam data retrievals not consistently using TeamHandler as source
```

Comment:
obsolete

Reply:
(This change comes from #954, I would instead suggest to review by commit and only consider the second one.)

Reply:
ah, i meant that it's not rebased onto the fresh version of #954 but i now see that #954 was never actually changed around this. i'll leave a note on #954